app/javascript/.storybook/preview.js

```diff
@@ -1,5 +1,52 @@
+import { h } from 'preact';
+import { addDecorator } from '@storybook/preact';
+
 import '../../assets/stylesheets/minimal.scss';
 import '../../assets/stylesheets/crayons.scss';
 import '../../assets/javascripts/lib/xss';
 import '../../assets/javascripts/utilities/timeAgo';
 import './storybook.scss';
+
+function changeTheme(theme) {
+  if (theme === '') {
+    return; // default theme
+  }
+
+  const head = document.head;
+  const link = document.createElement('link');
+
+  link.type = 'text/css';
+  link.rel = 'stylesheet';
+  link.href = `themes/${event.target.value}.css`;
+  link.id = 'dev-theme';
+
+  head.appendChild(link);
+}
+
+const themeSwitcher = (event) => {
+  const currentTheme = document.getElementById('dev-theme');
+
+  if (currentTheme) {
+    currentTheme.parentElement.removeChild(currentTheme);
+  }
+
+  changeTheme(event.target.value);
+};
+
+const themeSwitcherDecorator = (storyFn) => (
+  <div>
+    <label style={{ position: 'absolute', top: 0, left: 0, margin: '1rem' }}>
+      Theme{' '}
+      <select onChange={themeSwitcher}>
+        <option value="">Default</option>
+        <option value="minimal">Minimal</option>
+        <option value="hacker">Hacker</option>
+        <option value="night">Night</option>
+        <option value="pink">Pink</option>
```

Comment:
Can we have the same order as we have in `/setting/ux`?
- Default
- Night
- Minimal
- Pink
- Hacker